app/assets/src/components/visualizations/heatmap/Heatmap.js

```diff
@@ -8,6 +8,18 @@ import SvgSaver from "svgsaver";
 import symlog from "../../utils/d3/scales/symlog.js";
 import cs from "./heatmap.scss";
 import cx from "classnames";
+import { CategoricalColormap } from "../../utils/colormaps/CategoricalColormap.js";
```

Comment:
Would it be a lot of work to break this into small files? Similarly to how we talked about with React components. It'd be great to avoid these large files.

Reply:
(Can do this as follow-up)

Reply:
Yes, I plan to do that eventually. But would prefer not to do it on this round. Needs bit of thought.